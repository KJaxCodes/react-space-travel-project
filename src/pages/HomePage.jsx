
export default function HomePage() {
    return (
        <div className="home">
            <h2> Welcome, Commanders!</h2>

            <p className="home">
                In the not-so-distant future, Earth has fallen into ruin after centuries of environmental decline and neglect. But hope is not lost! Humanity has transformed other planets in our solar system into thriving, habitable worlds.
            </p>

            <p className="home">
                As a Commander, your mission is critical: evacuate the remaining population from Earth and lead them to safety using advanced spacecraft.
            </p>

            <h3>ðŸš€ What You Can Do:</h3>
            <ul>
                <li>View a list of available spacecraft and their current locations and detailed specs.</li>
                <li>Build a new spacecraft to aid in the evacuation efforts.</li>
                <li>Decommission old and damaged spacecraft.</li>
                <li>Explore planets and see what spacecraft are stationed there.</li>
                <li>View a list of available spacecraft and their current locations.</li>
                <li>Send spacecraft from one planet to another to safely relocate people.</li>
            </ul>

            <p className="home">
                This is your command center. Plan strategically, act swiftly, and lead humanity to a new beginning!
            </p>
        </div>
    )
}