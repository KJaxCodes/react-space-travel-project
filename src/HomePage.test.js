import React from "react";
import { render } from "@testing-library/react";
// components
import HomePage from "./pages/HomePage";

test("HomePage tests", () => {

    //expect(1).toBe(1);

    const { asFragment } = render(<HomePage />);

    expect(asFragment()).toMatchSnapshot();
})