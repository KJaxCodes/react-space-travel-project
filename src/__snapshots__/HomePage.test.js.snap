// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`HomePage tests 1`] = `
<DocumentFragment>
  <div
    class="home"
  >
    <h2>
       Welcome, Commanders!
    </h2>
    <p
      class="home"
    >
      In the not-so-distant future, Earth has fallen into ruin after centuries of environmental decline and neglect. But hope is not lost! Humanity has transformed other planets in our solar system into thriving, habitable worlds.
    </p>
    <p
      class="home"
    >
      As a Commander, your mission is critical: evacuate the remaining population from Earth and lead them to safety using advanced spacecraft.
    </p>
    <h3>
      ðŸš€ What You Can Do:
    </h3>
    <ul>
      <li>
        View a list of available spacecraft and their current locations and detailed specs.
      </li>
      <li>
        Build a new spacecraft to aid in the evacuation efforts.
      </li>
      <li>
        Decommission old and damaged spacecraft.
      </li>
      <li>
        Explore planets and see what spacecraft are stationed there.
      </li>
      <li>
        View a list of available spacecraft and their current locations.
      </li>
      <li>
        Send spacecraft from one planet to another to safely relocate people.
      </li>
    </ul>
    <p
      class="home"
    >
      This is your command center. Plan strategically, act swiftly, and lead humanity to a new beginning!
    </p>
  </div>
</DocumentFragment>
`;
